tools/mocha-runner/src/interface.ts

```diff
@@ -28,6 +32,9 @@ const skippedTests: Array<{testIdPattern: string; skip: true}> = process.env[
   ? JSON.parse(process.env['PUPPETEER_SKIPPED_TEST_CONFIG'])
   : [];
 
+const deflakeRetries = Number(process.env['PUPPETEER_DEFLAKE_RETRIES'] ?? 100);
+const deflakeTest: string | undefined = process.env['PUPPETEER_DEFLAKE_TEST'];
```

Comment:
Just make deflake and deflakeRetries one variable that takes `true` or a number when specified.

Reply:
I don't think this is an option as one `PUPPETEER_DEFLAKE_TEST` is the test pattern (See README) and the other is number of retries.

Reply:
Ah, okay. Hmm, let's make it `PUPPETEER_DEFLAKE_TESTS` then to clarify it's multiple tests